module/scripts/combat/effects.js

```diff
@@ -0,0 +1,94 @@
+// module\scripts\combat\effects.js
+export { applyBleed, applyFire, applyPoison, applySuffocation };
+
+async function changeHealth(actor, amount, name, remidy = "") {
+    // Ensure `amount` is a number
+    if (typeof amount !== 'number' || !isFinite(amount)) {
+      console.error(`Invalid amount for health change: ${amount}`);
+      return;
+    }
+  
+    const currentHP = actor.system.derivedStats.hp.value;
+    const maxHP = actor.system.derivedStats.hp.max;
+    const newHP = Math.min(Math.max(currentHP + amount, 0), maxHP);
+  
+    // Ensure the new HP is within valid range
+    if (newHP !== currentHP) {
+      await actor.update({
+        'system.derivedStats.hp.value': newHP
+      });
+  
+      if (amount > 0) {
+        ChatMessage.create({
+          content: `${actor.name} heals for ${amount} hit points because of ${name}!${remidy ? ` ${remidy}` : ''}`,
+          speaker: ChatMessage.getSpeaker({ actor: actor })
+        });
+      } else if (amount < 0) {
+        ChatMessage.create({
+          content: `${actor.name} takes ${-amount} hit points of ${name} damage!`,
+          speaker: ChatMessage.getSpeaker({ actor: actor })
+        });
+      }
+    } else {
+      console.log(`No change in HP detected for ${actor.name}.`);
+    }
+  }
+  
+  async function applyFireDamageToEquippedItems(actor, damage) {
+    // Update stopping power for equipped armor
+    for (const item of actor.items.filter(i => i.type === 'armor' && i.system.equipped)) {
+        const locationsCovered = ['head', 'torso', 'leftArm', 'rightArm', 'leftLeg', 'rightLeg'];
+        for (const location of locationsCovered) {
+            const stoppingField = `${location}Stopping`;
+            if (item.system[stoppingField] !== undefined) {
+                const stoppingPower = item.system[stoppingField] ?? 0;
+                if (stoppingPower > 0) {
+                    const newStoppingPower = Math.max(stoppingPower - damage, 0);
+                    await item.update({ [`system.${stoppingField}`]: newStoppingPower });
+                    ui.notifications.info(`${item.name} loses ${damage} stopping power in ${location}!`,
+                        { permanent: false, delay: 1000 });
+                }
+            }
+        }
+    }
+
+    // Update reliability for equipped weapons
+    for (const item of actor.items.filter(i => i.type === 'weapon' && i.system.equipped)) {
+        const reliabilityValue = item.system?.reliable ?? 0;
+        const newReliability = Math.max(reliabilityValue - 1, 0);
+        await item.update({ "system.reliable": newReliability });
+        ui.notifications.info(`${item.name} loses 1 reliability!`, { permanent: false, delay: 1000 });
+    }
+}
+
+async function applyFire(actor) {
+    const fireDamage = -5;
```

Comment:
This should be in config.js

Reply:
It might be that these do not translate into the active effect, then you would need to make a new subclass of it. If it comes to that I will help you

Reply:
I am trying to adjust the poison inside the config.js through adding flags:
flags: {
            combat: {
                turn: {
                    damageAmount: 3,
                    damage: {
                        location: 'torso',
                        damageProperties: {
                            bypassesNaturalArmor: true,
                            bypassesWornArmor: true,
                        },
                    }
                }
            }
        }

I noticed afterwards that I sadly could not access these through for example effect.combat.turn.damageAmount but just get an undefined error. Did I understand you wrong?
The new hook function I am trying to adjust the poison inside the config.js through adding flags:
flags: {
            combat: {
                turn: {
                    damageAmount: 3,
                    damage: {
                        location: 'torso',
                        damageProperties: {
                            bypassesNaturalArmor: true,
                            bypassesWornArmor: true,
                        },
                    }
                }
            }
        }

I noticed afterwards that I sadly could not access these through for example effect.combat.turn.damageAmount but just get an undefined error. Should it be done differently? Since I am still new to javascript (and foundry) my knowledge here is quite limited.

Reply:
No need for flags

I should just be 

```
{
        id: 'poison',
        name: 'WITCHER.statusEffects.poison',
        img: 'icons/svg/poison.svg',
        combat: {
                turn: {
                    damageAmount: 3,
                    damage: {
                        location: 'torso',
                        damageProperties: {
                            bypassesNaturalArmor: true,
                            bypassesWornArmor: true,
                        },
                    }
                }
            }
    },
```

Reply:
I will move it there. So dont worry about that now